### PR TITLE
bridge_ui: moved to webpack 5 and react-app-rewired, updated solana wallet deps

### PR DESCRIPTION
**Pull request by kev1n-peters**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/957)

### CODE REVIEW

File: bridge_ui/config-overrides.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    plugins: [
+      ...config.plugins,
+      new ProvidePlugin({
+        Buffer: ["buffer", "Buffer"],
+        process: "process/browser",
+      }),
+    ],
+    resolve: {
+      ...config.resolve,
+      fallback: {
```
Comment (kev1n-peters) [inline bridge_ui/config-overrides.js:30]: These are the polyfill packages that were used webpack 4: [Webpack 5 Node Polyfills Upgrade Cheatsheet
](https://gist.github.com/ef4/d2cf5672a93cf241fd47c020b9b3066a)

Comment (kev1n-peters) [inline bridge_ui/config-overrides.js:30]: These are the polyfill packages that were used webpack 4: [Webpack 5 Node Polyfills Upgrade Cheatsheet
](https://gist.github.com/ef4/d2cf5672a93cf241fd47c020b9b3066a)

**Review by evan-gray — APPROVED** [review]
> Reviewed 10 of 10 files at r1, all commit messages.
> Dismissed @kev1n-peters from a discussion.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/957)* status: all files reviewed, 2 unresolved discussions (waiting on @chase-45, @justinschuldt, and @kev1n-peters)

___
*[bridge_ui/src/components/SolanaConnectWalletDialog.tsx, line 76 at r1](https://reviewable.io/reviews/certusone/wormhole/957#-MyF7DJxCNw0K1fyYVi2:-MyF7DJxCNw0K1fyYVi3:b-oau0od) ([raw file](https://github.com/certusone/wormhole/blob/1ab398f9a7bcb67a472c241f66b9e267b801dd9d/bridge_ui/src/components/SolanaConnectWalletDialog.tsx#L76)):*
> ```XML
>   }, [wallets]);
> 
>   const handleWalletClick = useCallback(
> ```

Consider moving this and the whole ListItem for each detected wallet into it's own component (see below comment)
___
*[bridge_ui/src/components/SolanaConnectWalletDialog.tsx, line 98 at r1](https://reviewable.io/reviews/certusone/wormhole/957#-MyF7TI9DcGXhVmL9Mw_:-MyF7TI9DcGXhVmL9Mwa:bp1ek0o) ([raw file](https://github.com/certusone/wormhole/blob/1ab398f9a7bcb67a472c241f66b9e267b801dd9d/bridge_ui/src/components/SolanaConnectWalletDialog.tsx#L98)):*
> ```XML
>           <ListItem
>             button
>             onClick={() => handleWalletClick(wallet.adapter.name)}
> ```

This will recreate the function every time, if you move the useCallback into the functional component you can avoid this. (see above comment)

**Review by evan-gray — DISMISSED** [review]
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/957)* status: all files reviewed, 3 unresolved discussions (waiting on @chase-45, @justinschuldt, and @kev1n-peters)

___
*[bridge_ui/src/components/SolanaConnectWalletDialog.tsx, line 104 at r1](https://reviewable.io/reviews/certusone/wormhole/957#-MyF9E5GEY0p_xA1ix8E:-MyF9E5GEY0p_xA1ix8F:b-nvcf1f) ([raw file](https://github.com/certusone/wormhole/blob/1ab398f9a7bcb67a472c241f66b9e267b801dd9d/bridge_ui/src/components/SolanaConnectWalletDialog.tsx#L104)):*
> ```XML
>           </ListItem>
>         ))}
>         {undetected.map((wallet) => (
> ```

Might be worth a divider between detected and undetected (same for the Terra dialog) or some slight distinction?

**Review by evan-gray — APPROVED** [review]
:lgtm:

> Reviewed 6 of 6 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/957#-:-MyKD8-X-AUDp1z1aSWj:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @chase-45 and @justinschuldt)

**Review by chase-45 — APPROVED** [review]

**Pull request merged**